life_game/game.rb

```diff
@@ -0,0 +1,151 @@
+class Game
+attr_accessor :board
+SIZE = 30
+
+  def initialize(board)
+    @board = board
+  end
+
+  def self.input_board(input)
+    if input == nil
+      @board = r_board
+    else
+      @board = params_board(input)
+    end
+  end
+
+
+  def self.empty_board
+    Array.new(SIZE){Array.new(SIZE,0)}
+  end
+
+  def self.params_array(input)
+    i=0
+    params_array=[]
+    input.each do |cell|
+      if cell=="1"
+        cell=1
+      elsif input[i+1]=="OFF"||input[i+1]==nil
+          cell=0
+      else
+          cell=cell
+      end
+      params_array<<cell
+      i=i+1
+    end
+    params_array.delete("OFF")
+    params_array
+  end
+
+  def self.params_board(input)
+    params_board = Array.new(SIZE){Array.new}
+    i = 0
+    params_array = params_array(input)
+    params_board.each do |row|
+      params_array[i..i+SIZE-1].each do |cell|
+        row<<cell
+      end
+      i = i+SIZE
+    end
+    params_board
+  end
+
+  def self.r_board
```

Comment:
a może lepiej byłoby to zapisać w jakimś pliku np. `.json` i wczytywać podczas startu serwera (używając `JSON.parse(File.read('r_board.json'))` do stałej R_BOARD? i tutaj tylko wypisywać?